### PR TITLE
Add advisory for memory leak in ic-stable-structures

### PR DESCRIPTION
**Pull request by venkkatesh-sekar**

### CODE REVIEW

**Comment by venkkatesh-sekar** [comment]
Hi @Shnatsel, can you please have a look?

File: crates/ic-stable-structures/RUSTSEC-0000-0000.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+```toml
+[advisory]
+id = "RUSTSEC-0000-0000"
+package = "ic-stable-structures"
+date = "2023-05-21"
+url = "https://github.com/dfinity/stable-structures/pull/212"
+categories = ["memory-corruption"]
```
Comment (riking) [inline crates/ic-stable-structures/RUSTSEC-0000-0000.md:7]: The correct category is `denial-of-service`, nothing in the advisory text you've written suggests memory corruption?

File: crates/ic-stable-structures/RUSTSEC-0000-0000.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+keywords = ["canister", "icp", "stable memory"]
```
Comment (riking) [inline crates/ic-stable-structures/RUSTSEC-0000-0000.md:8]: maybe keyword "memory leak"?

File: crates/ic-stable-structures/RUSTSEC-0000-0000.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+```toml
+[advisory]
+id = "RUSTSEC-0000-0000"
+package = "ic-stable-structures"
+date = "2023-05-21"
```
Comment (riking) [inline crates/ic-stable-structures/RUSTSEC-0000-0000.md:5]: Date is 2024-05-17

**Review by riking — CHANGES_REQUESTED** [review]

File: crates/ic-stable-structures/RUSTSEC-0000-0000.md (hunk 1 above)
Comment (venkkatesh-sekar) [reply crates/ic-stable-structures/RUSTSEC-0000-0000.md]: Yeah, denial-of-service fits better.

**Comment by venkkatesh-sekar** [comment]
Hey @Shnatsel, friendly ping. Happy to discuss if something is blocking this.

**Review by kornelski — APPROVED** [review]

**Pull request merged**